### PR TITLE
fix dependencies with warning

### PR DESCRIPTION
**Pull request by delta-9**
I added the dependencies to resolve the issue in https://github.com/oliviertassinari/react-swipeable-views/issues/129

### CODE REVIEW

**Comment by oliviertassinari** [comment]
That looks good to me. Thanks for the fix.

File: package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
   "peerDependencies": {
     "react": "^15.0.0 || ^0.14.0",
     "react-dom": "^15.0.0 || ^0.14.0",
-    "react-motion": "^0.4.0"
+    "react-motion": "^0.4.0",
+    "warning": "^3.0.0"
```
Comment (oliviertassinari) [inline package.json]: What about moving it in the dependency section?
This modules shouldn't have any impact in production.

Comment (delta-9) [inline package.json]: Yeah you right, I do that

**Comment by delta-9** [comment]
Should be Ok now

**Pull request merged**